doc_source/stacksets-sampletemplates.md

```diff
@@ -1,6 +1,6 @@
 # AWS CloudFormation StackSets sample templates<a name="stacksets-sampletemplates"></a>
 
-This section includes links to some sample AWS CloudFormation templates that can help you use AWS CloudFormation StackSets in your enterprise\. Templates listed in this section enable [AWS Config](http://docs.aws.amazon.com/config/latest/developerguide/WhatIsConfig.html) and rules within it\.
+This section includes links to some sample AWS CloudFormation templates that can help you use AWS CloudFormation StackSets in your enterprise\. Templates listed in this section enable [CloudTrail](https://aws.amazon.com/cloudtrail) or [AWS Config](http://docs.aws.amazon.com/config/latest/developerguide/WhatIsConfig.html) and rules within it\.
```

Comment:
Hey @sot528 , I would love to merge your PR, could you change the name of CloudTrail to AWS CloudTrail. Thanks!

Reply:
And just for consistency, use this url: https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-user-guide.html

For the link to the AWS CloudTrail docs! Thanks.

Reply:
OK, Done. 👍